utils/features.py

```diff
@@ -0,0 +1,51 @@
+# Third Party (PyPI) Imports
+from dataclasses import dataclass
+
+# HTK Imports
+from apps.features.utils import is_feature_enabled
+
+# Django Imports
+from django.conf import settings
+
+# isort: off
+
+
+@dataclass
+class Feature:
+    name: str
+    label: str
+    description: str
+
+    @property
+    def is_enabled(self):
+        """Returns whether the feature is enabled."""
+        return is_feature_enabled(self.name)
+
+    def __str__(self):
+        return self.name
+
+    def __repr__(self):
+        return f'<Feature: {self.name} ({self.label})>'
+
+    def get_or_create(self):
+        """Gets or creates the FeatureFlag instance for this feature."""
+        module_path = f"{settings.PROJECT_NAME}.models"
+        FeatureFlag = __import__(module_path, fromlist=['FeatureFlag']).FeatureFlag
```

Comment:
There is already a util for getting the feature flag model.

It is `htk.apps.features.utils.get_feature_flag_model()`

```suggestion
        FeatureFlag = get_feature_flag_model()
```